policy/policy-data.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2015 ISRG.  All rights reserved
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package policy
+
+import (
+	"fmt"
+	"strings"
+
+	blog "github.com/letsencrypt/boulder/log"
+	"github.com/letsencrypt/boulder/sa"
+
+	gorp "github.com/letsencrypt/boulder/Godeps/_workspace/src/gopkg.in/gorp.v1"
+)
+
+const whitelisted = "whitelist"
+const blacklisted = "blacklist"
+
+// DomainRule ...
+type DomainRule struct {
+	Rule string `db:"rule"`
```

Comment:
This is sometimes referred to as "Rule" and sometimes referred to as "host". Please just call it Host if that's all it's doing right now. We can adjust the code in the future if we need to. Was there something specific you had in mind for this?